tests/test_models.py

```diff
@@ -513,3 +513,31 @@ def test_provider_name(self, mock_method):
         ec_idp = EnterpriseCustomerIdentityProviderFactory()
 
         assert ec_idp.provider_name == provider_name  # pylint: disable=no-member
+
+
+@mark.django_db
+@ddt.ddt
+class TestEnterpriseCustomerEntitlements(unittest.TestCase):
+    """
+    Tests of the TestEnterpriseCustomerEntitlements model.
+    """
+
+    @ddt.data(
+        str, repr
+    )
+    def test_string_conversion(self, method):
+        """
+        Test ``TestEnterpriseCustomerEntitlements`` conversion to string.
+        """
+        entitlement_id, enterprise_customer_name = 1234, "TestShib"
+        enterprise_customer = EnterpriseCustomerFactory(name=enterprise_customer_name)
+        ec_entitlements = EnterpriseCustomerEntitlement(
```

Comment:
It would be better to create `EnterpriseCustomerEntitlementFactory` this would shrink these two lines into one line something like this
```
EnterpriseCustomerEntitlementFactory(
    enterprise_customer__name= enterprise_customer_name, 
    entitlement_id=entitlement_id,
)
```